### PR TITLE
Switch from/to uuids around to match outer record

### PR DESCRIPTION
**Pull request by peteryates**
Small documentation fix, switch the from/to values around so the 'to' value matches the current one.

### CODE REVIEW

**Review by cwrw — APPROVED** [review]

File: docs/source/npq/guidance.html.md [hunk 1]
```diff
@@ -375,8 +375,8 @@ For more detailed information see the specifications for this [view multiple NPQ
         ],
         "participant_id_changes": [
           {
-            "from_participant_id": "ac3d1243-7308-4879-942a-c4a70ced400a",
-            "to_participant_id": "23dd8d66-e11f-4139-9001-86b4f9abcb02",
+            "from_participant_id": "23dd8d66-e11f-4139-9001-86b4f9abcb02",
```
Comment (cwrw) [inline docs/source/npq/guidance.html.md:378]: do you mind also updating the ECF guidance as well :)?

**Comment by cwrw** [comment]
also added for the spec https://github.com/DFE-Digital/early-careers-framework/pull/4437/files

**Comment by cwrw** [comment]
cherry-picked to https://github.com/DFE-Digital/early-careers-framework/pull/4437

**Comment by peteryates** [comment]
Fixed in #4437

**Pull request closed without merging**